### PR TITLE
Include the device tree for Raspberry Pi CM4S

### PR DESCRIPTION
**Pull request by chrblabla**
**- What I did**

- Added `(broadcom/)bcm2711-rpi-cm4s.dtb` to `RPI_KERNEL_DEVICETREE` as preparation for https://github.com/balena-os/balena-raspberrypi/pull/913.

### CODE REVIEW

**Review by agherzan — CHANGES_REQUESTED** [review]
Hi. Thanks for your contribution. A couple of comments:

1. We will also need the dt in https://github.com/agherzan/meta-raspberrypi/blob/master/conf/machine/raspberrypi-armv8.conf for the armv8 unified machine
2. Have a short look at https://meta-raspberrypi.readthedocs.io/en/latest/contributing.html. Especially around the Signed-off-by and I'd also reword it to be consistent with git history: `Include the device tree for Raspberry Pi CM4S`

**Comment by chrblabla** [comment]
Thank you for the quick reply, I'll update the PR shortly.

**Comment by chrblabla** [comment]
@agherzan should be good to go.

**Review by agherzan — APPROVED** [review]
